orchestrator/core/discoveryspace/samplers.py

```diff
@@ -42,6 +43,12 @@ class SamplerTypeEnum(enum.Enum):
     GENERATOR = "generator"
 
 
+class BaseParameters(pydantic.BaseModel):
```

Comment:
Maybe `BaseParameters` is too generic of a name - we could have it be `BaseSamplerParameters`, which  would be a consistent naming scheme with `BaseSamplerConfiguration`, although maybe the two might then risk being confused between one another